pocean/dsg/timeseries/om.py

```diff
@@ -232,7 +232,7 @@ def nc_attributes(self):
                 'long_name' : 'station identifier'
             },
             'time': {
-                'units': 'seconds since 1970-01-01T00:00:00Z',
+                'units': self.__class__.default_time_unit,
```

Comment:
You can use `self.default_time_unit` here since the OM class is a subclass of CF (where that is defined).